<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Attackwindow</class>
 <widget class="QDialog" name="Attackwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Side Channel Attack</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1" rowspan="3">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="DataSet">
       <property name="text">
        <string>Data Set</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="columnCount">
        <number>1</number>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column/>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DataButton">
       <property name="text">
        <string>Open Data Set</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ClearButton">
       <property name="text">
        <string>Clear Data Set</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="attackButton">
     <property name="text">
      <string>Launch Attack</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="methodLabel">
         <property name="text">
          <string>Attack Method</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="methodBox">
         <property name="currentText">
          <string/>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <item>
          <property name="text">
           <string>CPA</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Algorithm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="algoBox">
         <property name="currentText">
          <string/>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <item>
          <property name="text">
           <string>AES</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Selection Function</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="functionBox">
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <item>
          <property name="text">
           <string>1st SBOX output</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Points Range</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinpts_start"/>
       </item>
       <item>
        <widget class="QSpinBox" name="spinpts_end"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Number of traces</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinnb_traces"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Bytes Index</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="byteIdxTable">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::SolidLine</enum>
         </property>
         <property name="sortingEnabled">
          <bool>false</bool>
         </property>
         <property name="rowCount">
          <number>4</number>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <row/>
         <row/>
         <row/>
         <row/>
         <column/>
         <column/>
         <column/>
         <column/>
         <item row="0" column="0">
          <property name="text">
           <string>0</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item row="0" column="3">
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item row="1" column="2">
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item row="1" column="3">
          <property name="text">
           <string>13</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item row="2" column="2">
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item row="2" column="3">
          <property name="text">
           <string>14</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item row="3" column="1">
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item row="3" column="2">
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item row="3" column="3">
          <property name="text">
           <string>15</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
